docs/classifieds/index.md

```diff
@@ -32,17 +36,37 @@ Aragon is an open source, decentralized project fighting for the freedom to orga
+[Learn More](https://wiki.aragon.org/jobs/openings/dev-ops/) |
 
-[<h2>**Senior Solidity/EVM/Ethereum opening at Aragon**</h2>](https://wiki.aragon.one/jobs/openings/solidity/) |
+[<h2>**Documentation Technical Writer opening at Aragon**</h2>](https://wiki.aragon.org/jobs/openings/technical-writer/) |
 :-----------|
-We are looking for an Ethereum wizard to work with us on Aragon Core and the upcoming Aragon Network. |
-[Learn More](https://wiki.aragon.one/jobs/openings/solidity/) |
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world's worst problems. **We’re looking for a Technical Writer to help us communicate the different moving parts of Aragon Core and Dev Tools to facilitate the development of the Ethereum community**. Our Technical Writer will be the bridge between Aragon engineers and external developers, as well as the reference to support the growth of the Ethereum community through better documentation. |
+[Learn More](https://wiki.aragon.org/jobs/openings/technical-writer/) |
+
+[<h2>**UI Developer at Aragon One − Contract Position**</h2>](https://wiki.aragon.org/jobs/openings/ui-developer-contract/) |
+:-----------|
+**We are looking for a user interface developer to work with us on Aragon and the Aragon Apps**. Consider applying if you have the ability to ship high quality UI code that is maintainable, performant, accessible, precise, and compatible with modern browsers. |
+[Learn More](https://wiki.aragon.org/jobs/openings/ui-developer-contract/) |
+
+[<h2>**Technical Writer for the Aragon DAC**</h2>](https://wiki.aragon.org/jobs/openings/DAC-Technical_Writer/) |
+:-----------|
+Aragon is an open source, decentralized, project fighting for the freedom to organize. We believe that decentralized organizations can solve the world's worst problems. As part of the ongoing efforts to build out the Aragon infrastructure, the Aragon Foundation is collaborating with Giveth to build out a second core team that will help end-users experiment with the Aragon stack. **We're looking for Technical Writers to build out the documentation repository for supporting the user base**. |
+[Learn More](https://wiki.aragon.org/jobs/openings/DAC-Technical_Writer/) |
+
+[<h2>**UX Designer for Aragon DAC**</h2>](https://wiki.aragon.org/jobs/openings/DAC-UX_Designer/) |
```

Comment:
Broken table.
```suggestion
[<h2>**UX Designer for Aragon DAC**</h2>](https://wiki.aragon.org/jobs/openings/DAC-UX_Designer/) |
:-----------|
Aragon is an open source, decentralized, project fighting for the freedom to organize. We believe that decentralized organizations can solve the world's worst problems. As part of the ongoing efforts to build out the Aragon infrastructure, the Aragon Foundation is collaborating with Giveth to build out a second core team that will help end-users experiment with the Aragon stack. **We're looking for an experienced UX Designer to lead the creation of revolutionary governance modules**. The first task will be to research TCR's in order to design applications using reusable components from the Aragon UI toolset given the already designed contracts. The second task will be to redesign the Giveth DApp integrating the lessons we have learned from the Giveth Beta keeping in mind it must be easy to build using Aragon Apps. As in, what Aragon apps need to be built to so that the Giveth DApp can run natively off of Aragon, and what should these Apps look like. |
[Learn More](https://wiki.aragon.org/jobs/openings/DAC-UX_Designer/) |
```